quiche/src/lib.rs

```diff
@@ -1237,6 +1237,13 @@ impl Config {
         self.local_transport_params.disable_active_migration = v;
     }
 
+    /// Sets the congestion control algorithm used.
```

Comment:
Not sure why you moved this method up.  I think that we should prefer set_cc_algorithm_name over set_cc_algorithm

The reason is related to us introducing additional algorithm names as I am prototyping in https://github.com/cloudflare/quiche/commit/d4899cea6c5304703371b8df1308c73ec51203f8.  See also the comment below about `set_recovery_config_in_handshake`

Reply:
I moved it just for grouping the `set_cc_algorithm*` methods together. Given that `_name()` is implemented via converting string to the enum it just kinda made sense to put th enum method first.

As for the gcongestion stuff, why not just add new enum values rather than reuse the existing ones?